projects/burnr/src/components/ThemeHeader.tsx

```diff
@@ -4,12 +4,19 @@ import { makeStyles } from '@material-ui/core';
 
 const useStyles = makeStyles(theme => ({
 	root: {
-		alignItems: 'center',
+		position: 'fixed',
+		zIndex: -1,
```

Comment:
That moves the header below the `html`. `body` etc. elements and makes it unclickable. You can't switch the theme right now. It shouldn't be needed